interfaces/builtin/udisks2.go

```diff
@@ -375,51 +378,69 @@ func (iface *udisks2Interface) Name() string {
 func (iface *udisks2Interface) StaticInfo() interfaces.StaticInfo {
 	return interfaces.StaticInfo{
 		Summary:              udisks2Summary,
+		ImplicitOnClassic:    true,
 		BaseDeclarationSlots: udisks2BaseDeclarationSlots,
 	}
 }
 
 func (iface *udisks2Interface) DBusConnectedPlug(spec *dbus.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
-	spec.AddSnippet(udisks2ConnectedPlugDBus)
+	if !release.OnClassic {
+		spec.AddSnippet(udisks2ConnectedPlugDBus)
+	}
 	return nil
 }
 
 func (iface *udisks2Interface) DBusPermanentSlot(spec *dbus.Specification, slot *snap.SlotInfo) error {
-	spec.AddSnippet(udisks2PermanentSlotDBus)
+	if !release.OnClassic {
+		spec.AddSnippet(udisks2PermanentSlotDBus)
+	}
 	return nil
 }
 
 func (iface *udisks2Interface) AppArmorConnectedPlug(spec *apparmor.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
 	old := "###SLOT_SECURITY_TAGS###"
-	new := slotAppLabelExpr(slot)
+	var new string
```

Comment:
This is obviously fine but I was thinking it might be nice to teach slotAppLabelExpr to return "unconfined" when the slot is provided by the core snap.

Reply:
Ehh... maybe. I think for the time being I'd be explicit. We might encounter a situation where on some systems the service has system policy defined by the distro that we'll need to account for on a per interface basis. Until we see more of this (or even systems where the sysadmin has applied policy outside of the distro), I'd prefer to leave this as is. We'll probably want to do something, but not sure what yet.

Reply:
The suggestion wasn't meant for this branch. I agree we shouldn't touch it here.